docs/MANUAL.md

```diff
@@ -244,29 +244,35 @@ flags are described, and all mechanisms are listed, along with their enabled fun
 
 The following table lists the meaning of abbreviations:
 
-|abbreviation|corresponding function      |
- |------------|----------------------------|
-|`enc`       |Encryption                  |
-|`dec`       |Decryption                  |
-|`hsh`       |Hashing                     |
-|`sig`       |Signature                   |
-|`sir`       |Signature with recovery     |
-|`vfy`       |Verification                |
-|`vre`       |Verification with recovery  |
-|`gen`       |Key generation              |
-|`gkp`       |Key pair generation         |
-|`wra`       |Wrapping                    |
-|`unw`       |Unwrapping                  |
-|`der`       |Derivation                  |
+| abbreviation | corresponding function     |
+| ------------ | -------------------------- |
+| `enc`        | Encryption                 |
+| `dec`        | Decryption                 |
+| `hsh`        | Hashing                    |
+| `sig`        | Signature                  |
+| `sir`        | Signature with recovery    |
+| `vfy`        | Verification               |
+| `vre`        | Verification with recovery |
+| `gen`        | Key generation             |
+| `gkp`        | Key pair generation        |
+| `wra`        | Wrapping                   |
+| `unw`        | Unwrapping                 |
+| `der`        | Derivation                 |
 
 The last column tells whether the operation takes place inside the boundaries of the cryptographic module (`HW`) or at the
 library level (`SW`).
 
-Finally, for mechanisms supporting elliptic curve cryptography, there are additional capabilities printed:
-|abbreviation|capability meaning | |------------|------------------------------------------------| |`F^p`
-|Supports curves defined over prime-based fields | |`F^2m`      |Supports curves defined over power of 2 fields | |`par`
-|Supports custom parameters curves | |`nam`       |Supports well-known named curves | |`unc`       |Supports compressed
-points representation | |`cmp`       |Supports compressed points representation |
+Finally, for mechanisms supporting elliptic curve cryptography, there are
+additional capabilities printed:
+
+| abbreviation | capability meaning                              |
+| ------------ | ----------------------------------------------- |
+| `F^p`        | Supports curves defined over prime-based fields |
+| `F^2m`       | Supports curves defined over power of 2 fields  |
+| `par`        | Supports custom parameters curves               |
+| `nam`        | Supports well-known named curves                |
+| `unc`        | Supports compressed points representation       |
+| `cmp`        | Supports compressed points representation       |
```

Comment:
The descriptions for `unc` and `cmp` are the same.

I suspect `unc` should say:

> Supports **un**compressed points representation

Reply:
Hi @domodwyer, yes it should. Can you adjust your PR to reflect the change? As far as I can see it is left to "compressed". Many thanks!